openbas-api/src/main/java/io/openbas/rest/exercise/ExerciseApi.java

```diff
@@ -945,5 +947,22 @@ public List<Endpoint> endpoints(@PathVariable String exerciseId) {
   public List<Document> documents(@PathVariable String exerciseId) {
     return this.documentService.documentsForSimulation(exerciseId);
   }
+
+  @GetMapping(EXERCISE_URI + "/{exerciseId}/scenario")
+  @RBAC(
+      resourceId = "#exerciseId",
+      actionPerformed = Action.READ,
+      resourceType = ResourceType.SIMULATION)
+  @Operation(summary = "Get the Scenario linked to the exercise")
+  @ApiResponses(
+      value = {
+        @ApiResponse(responseCode = "200", description = "The Scenario"),
+        @ApiResponse(responseCode = "404", description = "Exercise or Scenario not found")
+      })
+  public Scenario scenarioFromExercise(
```

Comment:
scenarioFromExercise => scenarioFromSimulation
"ID of the exercise" => "ID of the simulation"
String exerciseId => String simulationId